UKHomeOffice/sgmr-service
# SPCR-460 Update README

### AC / Description of changes
From the description of [the SPCR-460 ticket](https://viabledata.atlassian.net/browse/SPCR-460)
> On the README for the sgmr-service repo there is a formatting issue at the end of the document. 
Text which is supposed to be in code blocks is formatted as normal text.

Formatting has been fixed and a missing link in the index has been added.
### To Test
* View README and compare to the one currently on master
* Check if formatting is now correct

### Notes

---

**Remember to merge to feature branches not `master`.**
